zeroex/order.go

```diff
@@ -88,7 +84,18 @@ type ContractEvent struct {
 	IsRemoved  bool
 	Address    common.Address
 	Kind       string
-	Parameters ContractEventParameters
+	Parameters interface{}
```

Comment:
@fabioberger changed this back to `interface{}` because for some event types, we just use the default `json.Marshal` behavior (which is more efficient and saves us some lines of code).